src/main/java/com/liferay/faces/util/context/internal/ExtFacesContextImpl.java

```diff
@@ -265,6 +266,26 @@ public void addMessage(FacesContext facesContext, String clientId, Severity seve
 		FacesContextHelperUtil.addMessage(facesContext, clientId, severity, messageId, arguments);
 	}
 
+	@Override
```

Comment:
@stiemannkj1 Are these methods necessary to add here? According to my IDE they don't seem to `@Override` anything?

Reply:
@stiemannkj1 Oh I see, ExtFacesContext implements FacesContextHelper.